subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt

```diff
@@ -11,7 +11,7 @@ public class CodeOwnershipValidationPlugin : Plugin<Project> {
         val strictOwnership = target.getBooleanProperty("avito.ownership.strictOwnership", false)
 
         target.afterEvaluate {
-            if (strictOwnership) {
+            if (strictOwnership && it.state.failure == null) {
```

Comment:
i have strongest deja-vu in my life that i already fixed it right on this line.
WTF ?:!)

Reply:
@dsvoronin not only you, WTF???

Reply:
https://github.com/avito-tech/avito-android/pull/1393/files

Reply:
Well, as I said: I forgot to include this change to a new plugin. Old plugin is working as intended. 
There was an issue with outdated local develop branch, I suppose. Now it's fixed, sorry for your trouble!